lib/screens/home/schedule.dart

```diff
@@ -4,7 +4,37 @@ class Schedule extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Text('Schedule'),
+      body: Center(
+        child: Container(
+          child: Column(
+            children: <Widget>[
+            boxOutline(),
+            Text("Monday 4-5pm"),
+            SizedBox(height: 7.5),
+            Text("Wednesday 5-6pm")
```

Comment:
It would be better if This part was in a ListView.builder() so that its more dynamic but otherwise it looks great!